buildSrc/build.gradle.kts

```diff
@@ -20,7 +23,17 @@ tasks.withType<KotlinCompile>().configureEach {
     }
 }
 
+gradlePlugin {
+    plugins {
+        create("keyappCoverage") {
+            id = "${group}.coverage"
+            implementationClass = "${group}.plugins.KeyAppCoverage"
+        }
+    }
+}
+
 dependencies {
+    implementation(gradleApi())
```

Comment:
does gradleApi provides "com.android.tools.build:gradle"?

Reply:
gradleApi() is internal dependency that comes directly from gradle dev kit, it provides gradle plugin development stuff